### ISSUE TITLE
[Accessibility: ACC-264.1] List item: selected state not read

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12563]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Use ngx [Combobox](https://sap.github.io/fundamental-ngx/#/core/combobox) component and try to read the selected value using screen reader, the selected state is not read.
Relationships and information conveyed by the presentation (such as labels) should be identified by assistive technologies so user can connect the accessed UI with associated label.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest versions.

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Use ngx [Combobox](https://sap.github.io/fundamental-ngx/#/core/combobox) component and try to read the selected value using screen reader, the selected state is not read.

#### Please provide relevant source code (if applicable).
NA

#### Please provide stackblitz example(s).
You can refer ngx example, it is reproducible. (https://stackblitz.com/run?file=src%2Fapp%2Fcombobox-example.component.ts)

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, issue is raised by an accessibility expert.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
@divyamalhotra488 the stackblitz links is incorrect.

**Comment by droshev** [comment]
closing as there was no activity
